.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,39 @@
+# Contributing
+
+If you discover issues, have ideas for improvements or new features,
+please report them to the [issue tracker][1] of the repository or
+submit a pull request. Please, try to follow these guidelines when you
+do so.
+
+## Issue reporting
+
+* Check that the issue has not already been reported.
+* Check that the issue has not already been fixed in the latest code
+  (a.k.a. `master`).
+* Be clear, concise and precise in your description of the problem.
+* Open an issue with a descriptive title and a summary in grammatically correct,
+  complete sentences.
+* Report the versions of `rubocop-rspec`, as well as the output of `rubocop -V`
+* Include any relevant code to the issue summary.
+
+## Pull requests
+1. Fork the project.
+2. Create a feature branch.
+3. Commit your changes.
+4. Make sure to add tests.
+5. Make sure the test suite is passing (run `rake`).
+6. Add [Changelog](../blob/master/CHANGELOG.md) entry.
+7. Push to the branch.
+8. Create new Pull Request.
+
+## Creating new cops
+There are some steps you have to follow when you add new cops:
+* Add an entry to `config/default.yml`. It's ordered list, make sure to follow the order.
+* The description of the cop in the code should match the one in config. `bin/build_config` takes care of this.
```

Comment:
It’s not clear from this description how `bin/build_config` takes care of this. Will the script copy the description over so I don’t need to do it manually, or does it just verify that they are identical?